index.html

```diff
@@ -785,6 +785,19 @@
           "site preferences".
         </p>
       </section>
+      <section>
+        <h3>
+          Managing source swapping between breakpoints
+        </h3>
+        <p>
+          When a new image is loaded via &lt;source&gt; after hitting a new
+          breakpoint, there is a delay while the image is loaded. Any CSS which
+          matches the breakpoint applies itself. However, we're stuck with the
+          previous image while the new one loads. This can result in blurry
```

Comment:
Can you `s/we're stuck with the previous image/the previous image still displays/`?